docs/concepts.ipynb

```diff
@@ -22,8 +22,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from scipp_widgets.input_spec import InputSpec\n",
-    "help(InputSpec)"
+    "from scipp_widgets.input import Input\n",
```

Comment:
Could we simply have `from scipp_widgets import Input`?

Reply:
We could one here is the module name and the other is a class. So there are several options.
 1. We an import the module and then lower down do `input.Input`
2. We could rename the class so it doesn't mirror the module name replacing. e.g. `Input` with `EvalInput` might make sense here.
3. We could rename the module to something else.

I'd be tempted to do 1 & 2 ?

Reply:
This section is reworked in the docs PR to link to the newly created list of existing inputs and provide a better example of how they work. I have updated the naming but don't think it is worth doing more on a bit that is removed for now.

Reply:
I was not thinking of renaming stuff, just importing `from input import Input` inside the `__init__.py` of `scipp-widgets`

Reply:
Ah ok would we want everything available on the Module or only the more commonly used things?

Reply:
I would say start with the commonly used things. We can always make more available later if users ask for it, but harder to remove once you made it available at some point.